quartz_solar_forecast/data.py

```diff
@@ -165,7 +165,8 @@ def make_pv_data(site: PVSite, ts: pd.Timestamp) -> xr.Dataset:
             site_id = matching_site_ids[0]
             live_generation_wh = get_solaredge_data(site_id)
     elif site.inverter_type == 'enphase':
-        live_generation_wh = get_enphase_data(ENPHASE_SYSTEM_ID)
+        # print("System ID: ", system_id)
```

Comment:
Should we remove this print comment for now?

Reply:
> Should we remove this print comment for now?

removed